libV2/schemaUtils.js

```diff
@@ -1240,6 +1243,10 @@ let QUERYPARAM = 'query',
         _.indexOf(requestBodySchema.required, key) !== -1;
       description = getParameterDescription(paramSchema);
 
+      if (encoding.hasOwnProperty(key) && encoding[key] && encoding[key].hasOwnProperty('contentType')) {
```

Comment:
Can use `_.get(encoding, `[${key}]contentType`)` to make sure to avoid type errors in cases where `encoding` is defined `null`.